smart-contracts/contracts/mixins/MixinLockManagerRole.sol

```diff
@@ -0,0 +1,47 @@
+pragma solidity ^0.5.0;
+
+import '@openzeppelin/upgrades/contracts/Initializable.sol';
+import '@openzeppelin/contracts-ethereum-package/contracts/access/Roles.sol';
+
+
+contract MixinLockManagerRole {
+  using Roles for Roles.Role;
+
+  event LockManagerAdded(address indexed account);
+  event LockManagerRemoved(address indexed account);
+
+  Roles.Role private _LockManagers;
```

Comment:
style consistency - shouldn't this be `lockManagers`?

Reply:
Yes, I've changed this to be consistent with our project style.